doc/tutorials/setup_db.ipynb

```diff
@@ -83,41 +88,59 @@
     "            \"BUCKETFS_PATH\": self.BUCKETFS_PATH\n",
     "        }\n",
     "\n",
-    "conf = SandboxConfig()"
+    "#conf = SandboxConfig()"
```

Comment:
Is this commented out code? If yes, please remove.

Reply:
That's intentional, to attract attention to changed usage of this notebook